atd-vze/src/views/Crashes/CR3Record.js

```diff
@@ -23,7 +23,16 @@ function CR3Record(props) {
     <Card>
       <CardHeader>Crash Report</CardHeader>
       <CardBody>
-        {props.isCr3Stored ? (
+        {props.isTempRecord ? (
+          <Alert color="warning">
+            <strong>CR3 PDFs are not available for temporary records.</strong><br />
+            Using the crash id, check the{" "}
```

Comment:
Should this be case ID instead of crash ID?

Reply:
Good call, it should be the case id!